myuw/data/forward_mapping.csv

```diff
@@ -0,0 +1,19 @@
+domain,login_url,link_title,icon_style
+deskmail.washington.edu,https://alpine.washington.edu,UW Deskmail,fa-envelope
+gamail.uw.edu,https://mail.google.com,UW Gmail,fa-google
```

Comment:
https://gmail.google.com will pre-select the uw login account - this could open any of their google acccounts